### PR TITLE
cleanup Phalcon\Session

### PR DESCRIPTION
**Pull request by mattvb91**
https://github.com/phalcon/cphalcon/issues/11340

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
 - `Phalcon\Acl\AdapterInterface::isAllowed` have 4th argument - parameters, you can pass arguments for function defined in `Phalcon\Acl\AdapterInterface:allow` or `Phalcon\Acl\AdapterInterface::deny` as associative array where key is argument name
 - Added method getActionSuffix() in `Phalcon\DispatcherInterface`
 - Added `Phalcon\Mvc\Controller\BindModelInterface` and associated model type hint loading through dispatcher.
+- Cleanup `Phalcon\Session` [#11340](https://github.com/phalcon/cphalcon/issues/11340)
```
Comment (sergeyklay) [inline CHANGELOG.md]: I think you should use `Removed` word

**Comment by virgofx** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (mattvb91) [inline CHANGELOG.md]: Updated :+1:

**Comment by Green-Cat** [comment]
Could you please rebase?

**Comment by andresgutierrez** [comment]
Could you please rebase?

**Comment by sergeyklay** [comment]
@mattvb91

**Comment by sergeyklay** [comment]
Fixed in `2.1.x` branch

**Pull request closed without merging**